Sources/HummingbirdCore/Request/RequestBody.swift

```diff
@@ -45,46 +48,48 @@ public enum HBRequestBody: Sendable, AsyncSequence {
     }
 }
 
-/// A type that represents an HTTP request body.
+/// Request body that is a stream of ByteBuffers.
 public struct HBStreamedRequestBody: Sendable, AsyncSequence {
     public typealias Element = ByteBuffer
+    public typealias InboundStream = NIOAsyncChannelInboundStream<HTTPRequestPart>
```

Comment:
At some point we are going to have to expose a public way to create a `HBRequestBody` that is initialised with a stream of data and at that time we'll have to expose the internal type to avoid having to do type conversions.